src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -861,6 +862,36 @@ def test_fuzzer_can_boot_and_run_reproducer(self):
                   result.output)
     self.assertIn('Running: data/fuchsia_crash', result.output)
 
+  @unittest.skipIf(
+      not environment.get_value('FUCHSIA_TESTS'),
+      'Temporarily disabling the Fuchsia tests until build size reduced.')
+  def test_qemu_logs_returned_on_error(self):
+    """Test running against a qemu that has died"""
+    test_helpers.patch(self, ['metrics.logs.log_warn'])
+    # Pass-through logs just so we can see what's going on (but moving from
+    # log_warn to plain log to avoid creating a loop)
+    self.mock.log_warn.side_effect = metrics.logs.log
+
+    environment.set_value('FUZZ_TARGET', 'example_fuzzers/overflow_fuzzer')
+    environment.set_value('JOB_NAME', 'libfuzzer_asan_fuchsia')
+    build_manager.setup_build()
+    testcase_path, _ = setup_testcase_and_corpus('fuchsia_crash',
+                                                 'empty_corpus')
+
+    runner = libfuzzer.FuchsiaQemuLibFuzzerRunner("fake/fuzzer")
```

Comment:
nit: please use single quotes for strings. another instance on lines 883 and 885.

Reply:
Done! Could this be made something that the linter reminds about?

Reply:
This was a limitation we encountered a while back with pylint :/ Not sure if this has changed unfortunately.